xample-malware/blackenergy.md

```diff
@@ -15,8 +15,41 @@ Behaviors
 ---------
 |Name|Use|
 |---|---|
-|[Denial of Service](../impact/denial-of-service.md)|Launches distributed denial of service attacks that can target more than one IP address per hostname. [[1]](#1)|
+|[Impact::Denial of Service (B0033)](../impact/denial-of-service.md)|Originally built to launch distributed denial of service attacks that can target more than one IP address per hostname  [[1]](#1)|
+|[Defense Evasion::Process Injection::Injection using Shims (E1055.m05)](../defense-evasion/process-injection.md)|Bypasses UAC using a Shim Database instructing SndVol.exe to execute cmd.exe instead, allowing for elevated execution  [[1]](#1)|
```

Comment:
There is corresponding software in ATT&CK, so under an "ATT&CK Techniques" heading should be a pointer to the ATT&CK link. This list should be labeled "MBC Behaviors."